lib/sse.js

```diff
@@ -8,6 +8,29 @@ let Promise = require('bluebird');
 let hl = require('highland');
 let JSONAPI_Error = require('./jsonapi-error');
 
+const Kafka = require('no-kafka');
+const brokers = "localhost:9092";
+
+const consumer = new Kafka.GroupConsumer({
+  connectionString: brokers,
+});
+
+const dataHandler = (req, res, harvesterInstance) => (messageSet, topic, partition) => {
+  return Promise.all(messageSet.map((m) => {
+    const doc = JSON.parse(m.message.value);
+    console.log('>>>> consumed from kafka >>', doc);
+
+    const eventName = inflect.pluralize('author') + '_' + doc.op;
+    //const id = doc.ts.getHighBits() + '_' + doc.ts.getLowBits();
+    const data = harvesterInstance.getData('author', doc);
+
+    tinySSE.send({ event: eventName, data: data })(req, res);
+    // commit offset
+    return consumer.commitOffset({topic: topic, partition: partition, offset: m.offset, metadata: 'optional'});
```

Comment:
We should commit the offset to mark the message kafka as read. If not, harvester will replay all the kafka messages if the harvester app restart for example.